Create an example OpenAPI spec (#5990)
kyma-project/kyma: docs/asset-store/03-04-supported-webhooks.md

  - return the `200` response with new file content.
  - return the `304` response informing that the file content was not modified.

- **validation service** must expose endpoints that:

  - contain **parameters** and **content** properties.
  - return the `200` response confirming that validation succeeded.
  - return the `422` response informing why validation failed.

- **metadata service** must expose endpoints that:

  - pass file data in the `"object": "string"` format in the request body, where **object** stands for the file name and **string** is the file content.
  - return the `200` response with extracted metadata.

> See the example of a service with the `/convert`, `/validate`, and `/extract` endpoints [here](./assets/example-openapi-service.yaml).

Reviewer: is is an example of a service or of a specification?
```suggestion
See the example of an API specification with the `/convert`, `/validate`, and `/extract` endpoints [here](./assets/example-openapi-service.yaml).
```
